G/GibbsSeaWater/build_tarballs.jl

```diff
@@ -30,9 +37,13 @@ products = [
     LibraryProduct("libgswteos", :libgswteos)
 ]
 
+llvm_version = v"13.0.1"
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    # We need libclang_rt.osx.a for linking libomp, because this library provides the
```

Comment:
The comment referring libomp doesn't make much sense here since there is no no libomp.